app/controllers/groups_controller.rb

```diff
@@ -405,6 +405,7 @@ def remove_members(member_ids_to_remove, assignment)
 
   # Removes the given student membership from the given grouping
   def remove_member(membership, grouping, assignment)
+    $stderr.puts "In remove_member"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.